composer.json

```diff
@@ -1,13 +1,13 @@
 {
     "name": "authorizenet/authorizenet",
     "type": "library",
-	"version": "2.0.2",
+	"version": "3.0.0",
     "description": "Official PHP SDK for Authorize.Net",
     "keywords": ["authorizenet", "authorize.net", "payment", "ecommerce"],
     "license": "proprietary",
     "homepage": "http://developer.authorize.net",
     "require": {
-        "php": ">=5.6",
+        "php": "^8.1",
```

Comment:
May I suggest removing this require? my guess is that previous versions of PHP will read the attribute as a comment, so no need to break compatibility